tests/components/vizio/test_media_player.py

```diff
@@ -561,9 +596,14 @@ async def test_setup_with_unknown_app_config(
     caplog: pytest.fixture,
 ) -> None:
     """Test device setup with apps where app config returned is unknown."""
-    await _test_setup_with_apps(
-        hass, MOCK_USER_VALID_TV_CONFIG, UNKNOWN_APP, UNKNOWN_APP_CONFIG
-    )
+    async with _cm_for_test_setup_tv_with_apps(
+        hass, MOCK_USER_VALID_TV_CONFIG, UNKNOWN_APP_CONFIG
+    ):
+        attr = hass.states.get(ENTITY_ID).attributes
+        _assert_source_list_with_apps(list(INPUT_LIST_WITH_APPS + APP_LIST), attr)
+        assert attr["source"] == UNKNOWN_APP
+        assert attr["app_name"] == UNKNOWN_APP
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5373293)